### DISCUSSION TITLE
Inquiries about posting the Kimai store plug-in

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4947] [Plugins]
Hello.
Can I sell a plug-in developed by an individual to the Kimai store for a fee?
Can paid plugins only be posted by Kimai? Can't individuals post? 
Are private developers only allowed to post free of charge?
Please reply.

**Comment by kevinpapst** [comment]
RTM: https://www.kimai.org/documentation/store.html

Yes you can. But you will have to send it to me first for a quality check.
